src/app/blocks/exception/exception.js

```diff
@@ -0,0 +1,21 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('blocks.exception')
+        .factory('exception', exception);
+
+    /* @ngInject */
```

Comment:
Do you need really need this? Doesn't ng-annotate handle this case? You're using a really old version of gulp-ng-annotate, try upgrading to the latest version and see if that helps